crates/re_viewer_context/src/selection_state.rs

```diff
@@ -117,7 +117,7 @@ pub struct SelectionState {
 impl SelectionState {
     /// Called at the start of each frame
     pub fn on_frame_start(&mut self, item_retain_condition: impl Fn(&Item) -> bool) {
-        re_tracing::profile_function!();
+        re_tracing::profile_scope!("SelectionState::on_frame_start");
```

Comment:
isn't that esactly what the profile_function macro prints, no?

Reply:
Not quite -- it seems like puffin doesn't track the struct name and so I was seeing two calls to `on_frame_start` coming from different call-sites resulting in some unexpected aggregation.

Reply:
Is that with latest puffin? afaik what scopes are merged when improved a bit